### DISCUSSION TITLE
firebase sms authentication not working - [auth/invalid-app-credential] Invalid Token

### DISCUSSION THREAD

**Discussion by fj-vp** [invertase/react-native-firebase#7540] [Q&A]
Hello dear community,

I'm facing the following issue:
Whenever I try to use phone authentication, the OTP code is not being received, due the error:
```
[auth/invalid-app-credential] Invalid Token
```
Here are some details about the project:
```
    "@react-native-firebase/app": "^14.12.0",
    "@react-native-firebase/auth": "^14.12.0",
    "@react-native-firebase/messaging": "^14.12.0"
...
```

The steps mentioned [here](https://rnfirebase.io/auth/phone-auth) are executed. Also:
* APN key created and uploaded in Firebase,
* Capability "background modes: Background fetch, remote notifications and background processing" are enabled,
* Capability "Push Notifications" is also enabled.

So far I'm running out of ideas what could have gone wrong. Until recently (i.e. 1-2 weeks ago) everything was working fine.

On Android everything works as expected.

Any help is appreciated!

**Comment by mikehardy** [comment]
Update all versions to current, post again if you still reproduce
